lib/renaissance/bids/bid.ex

```diff
@@ -21,8 +21,9 @@ defmodule Renaissance.Bid do
     bid
     |> cast(attrs, @required_fields ++ @optional_fields)
     |> validate_required(@required_fields)
-    |> foreign_key_constraint(:bidder_id, name: :bids_bidder_id_fkey)
-    |> foreign_key_constraint(:auction_id, name: :bids_auction_id_fkey)
+    |> foreign_key_constraint(:bidder_id)
+    |> foreign_key_constraint(:auction_id)
+    |> unique_constraint(:amount, name: :bids_amount_auction_id_index)
```

Comment:
do you NEED to specify the name of the index?

Reply:
Yes, the ecto docs says the following about [validating complex constraints](https://hexdocs.pm/ecto/Ecto.Changeset.html#unique_constraint/3-complex-constraints):  
> Because such indexes have usually more complex names, we need to explicitly tell the changeset which constraint name to use